modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -157,7 +162,10 @@ export default {
       errorCount: 0,
       restoreOnly: false,
       saveMenu: null,
-      generation: 0
+      generation: 0,
+      originalDoc: {
```

Comment:
The ref property should exist from the start, even if it's initially null. vue 2 reactivity can't detect a property being added for the first time (not without help). It can detect a property value change.

Reply:
@boutell You're right, I'll initialise `ref` 👍